### ISSUE TITLE
Unable to import module

### ISSUE THREAD

**Issue by saurabhgupta2u** [jdalrymple/node-gitlab#239]
---

**Description**

I'm trying to use this library in a React app to query data from my company's self hosted gitlab instance. However, after installing the library (npm install gitlab), I get an error when I try to import it.

node version: 8.13.0
npm version: 4.6.1
OS: Windows 10

**Steps to reproduce**
import Gitlab from "gitlab";

**Expected behaviour**

It should import without errors.

**Actual behaviour**

I get the following error in the browser (chrome v70) console:

```
TypeError: util_1.promisify is not a function
    at promisifyWithRetry (XMLHttpRequester.js?f3b1:19)
    at Object../node_modules/gitlab/dist/latest/infrastructure/XMLHttpRequester.js (XMLHttpRequester.js?f3b1:34)
    at __webpack_require__ (bootstrap 0771c941c50af96ff52f:686)
    at fn (bootstrap 0771c941c50af96ff52f:105)
    at Object../node_modules/gitlab/dist/latest/infrastructure/BaseService.js (BaseService.js?765c:8)
    at __webpack_require__ (bootstrap 0771c941c50af96ff52f:686)
    at fn (bootstrap 0771c941c50af96ff52f:105)
    at Object../node_modules/gitlab/dist/latest/infrastructure/index.js (index.js?cb7b:3)
    at __webpack_require__ (bootstrap 0771c941c50af96ff52f:686)
    at fn (bootstrap 0771c941c50af96ff52f:105)
    at Object../node_modules/gitlab/dist/latest/services/Groups.js (Groups.js?eb6b:3)
    at __webpack_require__ (bootstrap 0771c941c50af96ff52f:686)
    at fn (bootstrap 0771c941c50af96ff52f:105)
    at Object../node_modules/gitlab/dist/latest/services/index.js (index.js?e9da:4)
    at __webpack_require__ (bootstrap 0771c941c50af96ff52f:686)
    at fn (bootstrap 0771c941c50af96ff52f:105)
    at Object../node_modules/gitlab/dist/latest/index.js (index.js?0588:13)
    at __webpack_require__ (bootstrap 0771c941c50af96ff52f:686)
    at fn (bootstrap 0771c941c50af96ff52f:105)
    at Object.<anonymous> (index.js?bbe1:247)
    at Object../source/bustle/container/task_board/index.js (2.0771c941c50af96ff52f.hot-update.js:11403)
    at __webpack_require__ (bootstrap 0771c941c50af96ff52f:686)
```

Any ideas how to fix this?

**Comment by jdalrymple** [comment]
Hmm not sure, perhaps the utils library isnt being imported correctly inside the XMLHttpRequester.js file. Ill look into it :)

**Comment by r00ger** [comment]
@saurabhgupta2u  What is the version of `gitlab` you have installed?

**Comment by r00ger** [comment]
Oh, wait a second. The library is not supposed to be used in browser in the first place. For example, there are file operation performed in `/src/services/Projects.ts` which won't work in browser. However, it would be nice to limited support for browser usage.

@jdalrymple perhaps, another issue should be created. We should separate envs, narrow usage of node-specific API and gracefully fail on node-specific methods

**Comment by jdalrymple** [comment]
@r00ger I've removed the problematic functions in v5. @saurabhgupta2u Are you just looking for browser support or Xhr support? If its the fomer, perhaps test out the next release which has a browser friendly version of the library using the ky library instead of got.

**Comment by thearabbit** [comment]
I base on `Meteor JS`.
I got the same error
```js
TypeError: util_1.promisify is not a function
    at promisifyWithRetry (/node_modules/gitlab/dist/latest/infrastructure/XMLHttpRequester.js:27)
    at eval (/node_modules/gitlab/dist/latest/infrastructure/XMLHttpRequester.js:42)
    at dynamic-import.js?hash=80632d6551dcbf599f0fcc533ac8dcb3aef6eabe:138
    at fileEvaluate (modules-runtime.js?hash=d3c3e5d67c95f97a60888bda7373292efad3be5e:346)
    at Module.require (modules-runtime.js?hash=d3c3e5d67c95f97a60888bda7373292efad3be5e:248)
    at require (modules-runtime.js?hash=d3c3e5d67c95f97a60888bda7373292efad3be5e:268)
    at eval (/node_modules/gitlab/dist/latest/infrastructure/BaseService.js:16)
    at dynamic-import.js?hash=80632d6551dcbf599f0fcc533ac8dcb3aef6eabe:138
```

**Comment by thearabbit** [comment]
OK now work fine.
I use `Meteor.call` on client (don't use `import ........`)

**Comment by thearabbit** [comment]
How to solve to import this on `Meteor Client Side`???

**Comment by jdalrymple** [comment]
Have you tried importing the module via require? const Gitlab = require('gitlab') ?

**Comment by thearabbit** [comment]
I tried, still don't work
```js
// Server.js
const Gitlab = require('gitlab')

export const createIssue = new ValidatedMethod({
  name: 'demo.createIssue',
  mixins: [CallPromiseMixin],
  validate: new SimpleSchema({
    projectId: String,
    title: String,
    description: String,
  }).validator(),
  async run({ projectId, title, description }) {
    if (Meteor.isServer) {
      const api = new Gitlab({
        url: 'https://gitlab.com',
        token: '....',
      })

      let result = await api.Issues.create(projectId, {
         title,
         description,
      })

      return result
    }
  },
})
```
Get error
```js
TypeError: Gitlab is not a constructor
```

**Comment by thearabbit** [comment]
I tried 
```js
const { ProjectsBundle } = require('gitlab')
........
----------------
// Client call via import
import { createIssue } from '/imports/api/demos/gitlab'
```
Get the same error
```js
TypeError: util_1.promisify is not a function
    at promisifyWithRetry (/node_modules/gitlab/dist/latest/infrastructure/XMLHttpRequest
```

**Comment by jdalrymple** [comment]
Hmm, ill take a look this weekend! Hopefully have it back up and running for you soon!

**Comment by LongLiveCHIEF** [comment]
I just worked through this yesterday, need to use it like so:

```
const Gitlab = require('gitlab').default
```

**Comment by patrickalima98** [comment]
**Description**
Same probleme here, i trying use this library on my Vue Electron App, i am including this library with following code:

electron:  4.0.0
node-gitlab: next
OS: Windows 10

```js
const Gitlab = require('gitlab').default;
```

and i use on my function

```js
  private getGitlabConn() {
    const gitlab: any = new Gitlab({
      token: this.token, // Can be created in your profile.
    });

    gitlab.Users.current()
      .then((data: any) => {
        console.log(data);
      });
  }
```

In my console, i have this:
![image](https://user-images.githubusercontent.com/17622137/53687465-9ed5de80-3d13-11e9-911f-04866252c6a6.png)

Any ideas how to fix this?

**Comment by jdalrymple** [comment]
Try: const { Gitlab } = require('gitlab') in next. I added an example to the tests as well in the next branch for reference

**Comment by thearabbit** [comment]
I tried
```js
// Meteor Method
const { ProjectsBundle } = require('gitlab')
...
      const api = new ProjectsBundle({
        url: 'https://gitlab.com', // Defaults to https://gitlab.com
        token: '123456', // Can be created in your profile.
      })

      let result = await api.Issues.create(projectId, {
        title,
        description,
      })

      return result
```
Get the error
```js
I20190305-10:52:42.673(7)? Exception while invoking method 'demo.createIssue' { HTTPError: Response code 404 (Not Found)
I20190305-10:52:42.707(7)?     at EventEmitter.emitter.on (/Volumes/DATA/MeteorApp/meteor-element/node_modules/got/source/as-promise.js:74:19)
I20190305-10:52:42.707(7)?     at <anonymous>
I20190305-10:52:42.707(7)?     at process._tickDomainCallback (internal/process/next_tick.js:228:7)
I20190305-10:52:42.708(7)?  => awaited here:
I20190305-10:52:42.708(7)?     at Function.Promise.await (/Users/theara/.meteor/packages/promise/.0.11.2.qgwvxk.djn5++os+web.browser+web.browser.legacy+web.cordova/npm/node_modules/meteor-promise/promise_server.js:56:12)
I20190305-10:52:42.708(7)?     at Promise.asyncApply (imports/api/demos/gitlab.js:22:17)
I20190305-10:52:42.708(7)?     at /Users/theara/.meteor/packages/promise/.0.11.2.qgwvxk.djn5++os+web.browser+web.browser.legacy+web.cordova/npm/node_modules/meteor-promise/fiber_pool.js:43:40
I20190305-10:52:42.708(7)?   name: 'HTTPError',
I20190305-10:52:42.708(7)?   host: undefined,
I20190305-10:52:42.708(7)?   hostname: 'gitlab.com',
I20190305-10:52:42.709(7)?   method: 'POST',
I20190305-10:52:42.709(7)?   path: '/api/v4/https://gitlab.com/projects/10852033/issues',
I20190305-10:52:42.709(7)?   socketPath: undefined,
I20190305-10:52:42.709(7)?   protocol: 'https:',
I20190305-10:52:42.709(7)?   url: 'https://gitlab.com/api/v4/https://gitlab.com/projects/10852033/issues',
I20190305-10:52:42.709(7)?   gotOptions:
I20190305-10:52:42.709(7)?    { path: '/api/v4/https://gitlab.com/projects/10852033/issues',
I20190305-10:52:42.712(7)?      protocol: 'https:',
I20190305-10:52:42.712(7)?      hostname: 'gitlab.com',
I20190305-10:52:42.712(7)?      hash: '',
I20190305-10:52:42.712(7)?      search: '',
I20190305-10:52:42.712(7)?      pathname: '/api/v4/https://gitlab.com/projects/10852033/issues',
I20190305-10:52:42.712(7)?      href: 'https://gitlab.com/api/v4/https://gitlab.com/projects/10852033/issues',
I20190305-10:52:42.712(7)?      retry:
I20190305-10:52:42.713(7)?       { retries: [Function],
I20190305-10:52:42.713(7)?         methods: [Object],
I20190305-10:52:42.713(7)?         statusCodes: [Object],
I20190305-10:52:42.713(7)?         errorCodes: [Object] },
I20190305-10:52:42.713(7)?      headers:
I20190305-10:52:42.713(7)?       { 'user-agent': 'got/9.6.0 (https://github.com/sindresorhus/got)',
I20190305-10:52:42.713(7)?         'private-token': '4ykTNcsNnw9scnDFXAhv',
I20190305-10:52:42.713(7)?         accept: 'application/json',
I20190305-10:52:42.714(7)?         'accept-encoding': 'gzip, deflate',
I20190305-10:52:42.714(7)?         'content-type': 'application/json',
I20190305-10:52:42.714(7)?         'content-length': 36 },
I20190305-10:52:42.714(7)?      hooks:
I20190305-10:52:42.714(7)?       { beforeRequest: [],
I20190305-10:52:42.714(7)?         beforeRedirect: [],
I20190305-10:52:42.714(7)?         beforeRetry: [],
I20190305-10:52:42.714(7)?         afterResponse: [],
I20190305-10:52:42.714(7)?         beforeError: [],
I20190305-10:52:42.715(7)?         init: [] },
I20190305-10:52:42.715(7)?      decompress: true,
I20190305-10:52:42.715(7)?      throwHttpErrors: true,
I20190305-10:52:42.715(7)?      followRedirect: true,
I20190305-10:52:42.715(7)?      stream: false,
I20190305-10:52:42.715(7)?      form: false,
I20190305-10:52:42.715(7)?      json: true,
I20190305-10:52:42.715(7)?      cache: false,
I20190305-10:52:42.716(7)?      useElectronNet: false,
I20190305-10:52:42.716(7)?      baseUrl: [Getter/Setter],
I20190305-10:52:42.716(7)?      body: '{"title":"Test","description":"adf"}',
I20190305-10:52:42.716(7)?      rejectUnauthorized: true,
I20190305-10:52:42.716(7)?      method: 'POST' },
I20190305-10:52:42.716(7)?   statusCode: 404,
I20190305-10:52:42.716(7)?   statusMessage: 'Not Found',
I20190305-10:52:42.717(7)?   headers:
I20190305-10:52:42.717(7)?    { server: 'nginx',
I20190305-10:52:42.717(7)?      date: 'Tue, 05 Mar 2019 03:52:42 GMT',
I20190305-10:52:42.717(7)?      'content-type': 'application/json',
I20190305-10:52:42.717(7)?      'content-length': '25',
I20190305-10:52:42.718(7)?      connection: 'close',
I20190305-10:52:42.718(7)?      'cache-control': 'no-cache',
I20190305-10:52:42.718(7)?      vary: 'Origin',
I20190305-10:52:42.718(7)?      'x-request-id': '0aiecg4W3K',
I20190305-10:52:42.718(7)?      'x-runtime': '0.006219',
I20190305-10:52:42.718(7)?      'ratelimit-limit': '600',
I20190305-10:52:42.719(7)?      'ratelimit-observed': '1',
I20190305-10:52:42.719(7)?      'ratelimit-remaining': '599',
I20190305-10:52:42.719(7)?      'ratelimit-reset': '1551758022',
I20190305-10:52:42.719(7)?      'ratelimit-resettime': 'Wed, 05 Mar 2019 03:53:42 GMT' },
I20190305-10:52:42.719(7)?   body: { error: '404 Not Found' } }
```

**Comment by thearabbit** [comment]
And then tried to remove `async/await`.
```js
(node:1680) UnhandledPromiseRejectionWarning: HTTPError: Response code 404 (Not Found)
W20190305-10:56:57.391(7)? (STDERR)     at EventEmitter.emitter.on (/Volumes/DATA/MeteorApp/meteor-element/node_modules/got/source/as-promise.js:74:19)
W20190305-10:56:57.391(7)? (STDERR)     at <anonymous>
W20190305-10:56:57.392(7)? (STDERR)     at process._tickDomainCallback (internal/process/next_tick.js:228:7)
W20190305-10:56:57.392(7)? (STDERR) (node:1680) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
W20190305-10:56:57.392(7)? (STDERR) (node:1680) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by casz** [comment]
@thearabbit the error is pretty clear. HTTP Error 404 means not found. In this case it did not find the project. So import is working

**Comment by thearabbit** [comment]
But I tried on `Stable Version`, it work fine.

**Comment by jdalrymple** [comment]
Yea, ill keep looking. Thanks

**Comment by jdalrymple** [comment]
@thearabbit could you provide me with any more information? I created three extra tests to validate the bundle and the issue creation work and they seem to not be running into this problem.

1. What version of the library are you using, you said next, but which next version? 
2. For your example, and its output, it looked like the paths were being mangled. Using your coding example i haven't been able to create the same issue however. This is the test ive been using to try and recreate the problem: https://github.com/jdalrymple/node-gitlab/blob/next-%23239/test/integration/bundles/ProjectsBundle.ts#L15

**Comment by jdalrymple** [comment]
@patrickalima98  did this help?

 > Try: const { Gitlab } = require('gitlab') in next. I added an example to the tests as well in the next branch for reference

**Comment by patrickalima98** [comment]
@jdalrymple sorry! I forgot to update here, yes this code fix this error, but now i am waiting for the correction of #282

**Comment by thearabbit** [comment]
@jdalrymple, sorry for late to reply.
I tried `"gitlab": "^5.0.0-rc.9"`

**Comment by jdalrymple** [comment]
@thearabbit hmm, and you get that error eh? Could it be related to the meteor environment some how? If you test it in a basic node script do you get the same error?

**Comment by jdalrymple** [comment]
@thearabbit could you perhaps provide a sample project?

**Comment by jdalrymple** [comment]
@thearabbit Gonna close this. If you can check if it happens with meteor only reopen and post!

**Comment by thearabbit** [comment]
@jdalrymple, sorry.
this is my repo `Meteor + Vue + Node-Gitlab`
https://github.com/thearabbit/meteor-gitlab

**Comment by jdalrymple** [comment]
@thearabbit perfect! Ill give it a look in the next couple days

**Comment by thearabbit** [comment]
Hi @jdalrymple, how about this?

**Comment by jdalrymple** [comment]
@thearabbit The project id you specified isnt linked to any project on gitlab. Look at my fork, it ran correctly, but the error was related to the project not existing.

**Comment by thearabbit** [comment]
@jdalrymple, this is my gitlab repo testing

![image](https://user-images.githubusercontent.com/4495160/56939151-df45b300-6b30-11e9-9633-41c7d7c72f4e.png)

But get error

```
{ HTTPError: Response code 404 (Not Found)
I20190430-10:13:59.607(7)?     at EventEmitter.emitter.on (/Volumes/DATA/MeteorApp/meteor-gitlab/node_modules/got/source/as-promise.js:74:19)
I20190430-10:13:59.609(7)?     at <anonymous>
I20190430-10:13:59.609(7)?     at process._tickDomainCallback (internal/process/next_tick.js:229:7)
I20190430-10:13:59.609(7)?   name: 'HTTPError',
I20190430-10:13:59.609(7)?   host: undefined,
I20190430-10:13:59.610(7)?   hostname: 'gitlab.com',
I20190430-10:13:59.610(7)?   method: 'POST',
I20190430-10:13:59.610(7)?   path: '/api/v4/https://gitlab.com/projects/10852033/issues',
I20190430-10:13:59.610(7)?   socketPath: undefined,
I20190430-10:13:59.610(7)?   protocol: 'https:',
I20190430-10:13:59.610(7)?   url: 'https://gitlab.com/api/v4/https://gitlab.com/projects/10852033/issues',
I20190430-10:13:59.610(7)?   gotOptions:
I20190430-10:13:59.610(7)?    { path: '/api/v4/https://gitlab.com/projects/10852033/issues',
I20190430-10:13:59.611(7)?      protocol: 'https:',
I20190430-10:13:59.611(7)?      hostname: 'gitlab.com',
I20190430-10:13:59.611(7)?      hash: '',
I20190430-10:13:59.611(7)?      search: '',
I20190430-10:13:59.611(7)?      pathname: '/api/v4/https://gitlab.com/projects/10852033/issues',
I20190430-10:13:59.611(7)?      href: 'https://gitlab.com/api/v4/https://gitlab.com/projects/10852033/issues',
I20190430-10:13:59.650(7)?      retry:
I20190430-10:13:59.651(7)?       { retries: [Function],
I20190430-10:13:59.651(7)?         methods: [Object],
I20190430-10:13:59.652(7)?         statusCodes: [Object],
I20190430-10:13:59.652(7)?         errorCodes: [Object] },
I20190430-10:13:59.652(7)?      headers:
I20190430-10:13:59.653(7)?       { 'user-agent': 'got/9.6.0 (https://github.com/sindresorhus/got)',
I20190430-10:13:59.653(7)?         'private-token': '4ykTNcsNnw9scnDFXAhv',
I20190430-10:13:59.653(7)?         accept: 'application/json',
I20190430-10:13:59.653(7)?         'accept-encoding': 'gzip, deflate',
I20190430-10:13:59.653(7)?         'content-type': 'application/json',
I20190430-10:13:59.654(7)?         'content-length': 45 },
I20190430-10:13:59.654(7)?      hooks:
I20190430-10:13:59.654(7)?       { beforeRequest: [],
I20190430-10:13:59.654(7)?         beforeRedirect: [],
I20190430-10:13:59.655(7)?         beforeRetry: [],
I20190430-10:13:59.655(7)?         afterResponse: [],
I20190430-10:13:59.664(7)?         beforeError: [],
I20190430-10:13:59.664(7)?         init: [] },
I20190430-10:13:59.664(7)?      decompress: true,
I20190430-10:13:59.665(7)?      throwHttpErrors: true,
I20190430-10:13:59.665(7)?      followRedirect: true,
I20190430-10:13:59.665(7)?      stream: false,
I20190430-10:13:59.665(7)?      form: false,
I20190430-10:13:59.665(7)?      json: true,
I20190430-10:13:59.666(7)?      cache: false,
I20190430-10:13:59.666(7)?      useElectronNet: false,
I20190430-10:13:59.666(7)?      baseUrl: [Getter/Setter],
I20190430-10:13:59.666(7)?      body: '{"title":"Title","description":"Description"}',
I20190430-10:13:59.666(7)?      rejectUnauthorized: true,
I20190430-10:13:59.666(7)?      method: 'POST' },
I20190430-10:13:59.666(7)?   statusCode: 404,
I20190430-10:13:59.667(7)?   statusMessage: 'Not Found',
I20190430-10:13:59.667(7)?   headers:
I20190430-10:13:59.667(7)?    { server: 'nginx',
I20190430-10:13:59.667(7)?      date: 'Tue, 30 Apr 2019 03:13:59 GMT',
I20190430-10:13:59.667(7)?      'content-type': 'application/json',
I20190430-10:13:59.667(7)?      'content-length': '25',
I20190430-10:13:59.667(7)?      connection: 'close',
I20190430-10:13:59.668(7)?      'cache-control': 'no-cache',
I20190430-10:13:59.668(7)?      vary: 'Origin',
I20190430-10:13:59.668(7)?      'x-request-id': 'rFUuyZWBCy8',
I20190430-10:13:59.668(7)?      'x-runtime': '0.008663',
I20190430-10:13:59.668(7)?      'ratelimit-limit': '600',
I20190430-10:13:59.668(7)?      'ratelimit-observed': '1',
I20190430-10:13:59.669(7)?      'ratelimit-remaining': '599',
I20190430-10:13:59.669(7)?      'ratelimit-reset': '1556594099',
I20190430-10:13:59.669(7)?      'ratelimit-resettime': 'Wed, 30 Apr 2019 03:14:59 GMT' },
I20190430-10:13:59.669(7)?   body: { error: '404 Not Found' } }
```

**Comment by thearabbit** [comment]
But it is `Private Repo`

**Comment by thearabbit** [comment]
Now I tried create new repo `Public`, but get error the same.
Could you try your testing repo?

**Comment by jdalrymple** [comment]
@thearabbit Yea i was just able to recreate it. Not sure why its happening. I've tested the call outside of meteor and it works fine. For some reason the url used in the request is being changed when run through meteor. Im looking.

**Comment by thearabbit** [comment]
thanks for your reply. We should issue this on `Meteor Forums` or not?

**Comment by jdalrymple** [comment]
lmao. Found the problem. Not sure how I missed this. The error has to do with the API [change](https://github.com/jdalrymple/node-gitlab/tree/next) with version 5. To avoid confusion with url and host, the url keyword is no longer user and is reserved for internal processes. Unfortunately, its still accessible so in your sample, both the host and url are set, which causes the wrong path to be used when making requests. Ill see if i can change it so its not externally exposed

**Comment by jdalrymple** [comment]
@thearabbit I made the changes in the fix/exposed-base-service branch. It will be merged into next once #265 is finished

**Comment by thearabbit** [comment]
Hi @jdalrymple, could I try this?
I updated to `"gitlab": "^6.0.0"`, but still get error
```js
import { Meteor } from 'meteor/meteor'
import { ValidatedMethod } from 'meteor/mdg:validated-method'
import { CallPromiseMixin } from 'meteor/didericis:callpromise-mixin'
import SimpleSchema from 'simpl-schema'
import { ProjectsBundle } from 'gitlab'

export const createIssue = new ValidatedMethod({
  name: 'demo.createIssue',
  mixins: [CallPromiseMixin],
  validate: new SimpleSchema({
    projectId: String,
    title: String,
    description: String,
  }).validator(),
  run({ projectId, title, description }) {
    if (Meteor.isServer) {
      const services = new ProjectsBundle({
        url: 'https://gitlab.com', // Defaults to https://gitlab.com
        token: '.....', // Can be created in your profile.
      })

      services.Issues.create(projectId, {
        title,
        description,
      })
        .then(res => {
          console.log(res)
        })
        .catch(err => {
          console.log(err)
        })

      return 'result'
    }
  },
})
```
Error
```js
{ HTTPError: Not Found
I20190605-11:50:37.076(7)?     at fn (/Volumes/DATA/MeteorApp/meteor-element/node_modules/ky/umd.js:242:12)
I20190605-11:50:37.076(7)?     at <anonymous>
I20190605-11:50:37.076(7)?     at process._tickDomainCallback (internal/process/next_tick.js:228:7)
I20190605-11:50:37.076(7)?   name: 'HTTPError',
I20190605-11:50:37.077(7)?   response:
I20190605-11:50:37.077(7)?    Response {
I20190605-11:50:37.077(7)?      size: 0,
I20190605-11:50:37.077(7)?      timeout: 0,
I20190605-11:50:37.077(7)?      [Symbol(Body internals)]: { body: [Object], disturbed: true, error: null },
I20190605-11:50:37.077(7)?      [Symbol(Response internals)]:
I20190605-11:50:37.077(7)?       { url: 'https://gitlab.com/api/v4/https:/gitlab.com/projects/10852033/issues',
I20190605-11:50:37.077(7)?         status: 404,
I20190605-11:50:37.078(7)?         statusText: 'Not Found',
I20190605-11:50:37.078(7)?         headers: [Object],
I20190605-11:50:37.078(7)?         counter: 1 } },
I20190605-11:50:37.078(7)?   description: '404 Not Found' }
```

**Comment by casz** [comment]
You used url again, when it should be host.
Please look at the Readme examples once again.

**Comment by thearabbit** [comment]
Very thanks, Now work fine
